content/docs/user-guide/components/reference/gradients/reference-gradient.md

```diff
@@ -0,0 +1,52 @@
+---
+linktitle: Reference Gradient
+title: Reference Gradient Component
+description: Use the Reference Gradient component to reference and re-use gradients in Open 3D Engine (O3DE).
+---
+
+Add the **Reference Gradient** component to reference and re-use gradients in your **Open 3D Engine (O3DE)** level.
+
+## Provider
+
+[Gradient Signal Gem](/docs/user-guide/gems/reference/utility/gradient-signal)
+
+## Reference Gradient properties
+
+![Reference Gradient component properties](/images/user-guide/components/reference/gradients/reference-gradient-component.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Preview** | Displays the output gradient of this component after all properties are applied. | | |
+| **Pin Preview to Shape** | Sets an entity with a compatible Shape component to use as the bounds of the preview if **Constrain to Shape** is `Enabled`. | EntityId | Current Entity |
+| **Preview Position** | Sets the world location of the preview.<br> <br>*This field is available only if there is no entity selected in **Pin Preview to Shape**.* | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
+| **Preview Size** | Sets the dimensions of the preview. | Vector3: 0.0 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
+| **Constrain to Shape** | If `Enabled`, the gradient preview will use the bounds of the entity selected in **Pin Preview to Shape**.<br> <br>*This field is available only if an entity is selected in **Pin Preview to Shape**.* | Boolean | `Disabled` |
+
+### Gradient properties
+
+![Gradient properties](/images/user-guide/components/reference/vegetation-modifiers/gradient-properties.png)
```

Comment:
Link is pointing to vegetation-modifiers directory and the image doesn't seem to be in this PR.

```suggestion
![Gradient properties](/images/user-guide/components/reference/gradients/gradient-properties.png)
```

Reply:
Yeah, I am reusing the image intentionally here, it could move to the static directory if need be.

Reply:
> Yeah, I am reusing the image intentionally here, it could move to the static directory if need be.

Oh! Sorry about that. I didn't think to check for it outside of this PR. It didn't render in the preview and I assumed it was an incorrect link.